prow/config.yaml

```diff
@@ -291,6 +291,16 @@ branch-protection:
         contexts:
         - cla/linuxfoundation
       repos:
+        cluster-api-provider-digitalocean:
+          required_status_checks:
+            contexts:
+            - "ci/circleci: build"
```

Comment:
circle ci was enabled for `kubermatic/cluster-api-digital-ocean-provider`, but it's not enabled for `kubernetes-sigs/...`. An admin will need to enable this first.

/hold

Reply:
We have lost the deploy key while transitioning to the new repository, so CircleCI was broken. I created a new deploy key and tests are now green.

/hold cancel